amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/optimizing/OptimizingStatus.java

```diff
@@ -19,20 +19,23 @@
 package org.apache.amoro.server.optimizing;
 
 public enum OptimizingStatus {
-  FULL_OPTIMIZING("full", true),
-  MAJOR_OPTIMIZING("major", true),
-  MINOR_OPTIMIZING("minor", true),
-  COMMITTING("committing", true),
-  PLANNING("planning", false),
-  PENDING("pending", false),
-  IDLE("idle", false);
+  FULL_OPTIMIZING("full", true, 0),
+  MAJOR_OPTIMIZING("major", true, 1),
+  MINOR_OPTIMIZING("minor", true, 2),
+  COMMITTING("committing", true, 3),
+  PLANNING("planning", false, 4),
+  PENDING("pending", false, 5),
+  IDLE("idle", false, 6);
```

Comment:
I suggest using non-contiguous codes for definitions to make future expansions more convenient.